pkg/cqrs/sqlitecqrs/migrations/000001_create_apps_table.up.sql

```diff
@@ -141,5 +141,6 @@ CREATE TABLE trace_runs (
 	output BLOB,
 	is_debounce BOOLEAN NOT NULL,
 	batch_id BLOB,
-	cron_schedule TEXT
+	cron_schedule TEXT,
+	has_ai BOOLEAN NOT NULL DEFAULT FALSE
```

Comment:
We'll need a new migration for this instead of editing the old one.

Reply:
new migration added